presto-plugin-toolkit/src/test/java/io/prestosql/plugin/base/security/TestFileBasedSystemAccessControl.java

```diff
@@ -268,4 +314,14 @@ private String getResourcePath(String resourceName)
     {
         return this.getClass().getClassLoader().getResource(resourceName).getPath();
     }
+
+    private static void assertDenied(ThrowingRunnable runnable)
+    {
+        assertThrows(AccessDeniedException.class, runnable);
+    }
+
+    private static void assertDeniedWithIllegalArgumentException(ThrowingRunnable runnable)
+    {
+        assertThrows(IllegalArgumentException.class, runnable);
```

Comment:
can you use assertj. `assertThatThrownBy()` and assert error message here?

Reply:
@kokosing - This is done